modules/counter/server-ts/index.ts

```diff
@@ -7,5 +7,5 @@ import ServerModule from '@gqlapp/module-server-ts';
 export default new ServerModule({
   schema: [schema],
   createResolversFunc: [createResolvers],
-  createContextFunc: [() => ({ Counter: new Counter() })]
+  createGraphQLContextFunc: [() => ({ Counter: new Counter() })]
```

Comment:
Theoretically this is a good change, but practically it is breaking change. You are forcing all existing projects based on the kit to do this renaming in their modules, when they upgrade. Is it worth it?

Reply:
Done